packages/source/types/data.ts

```diff
@@ -49,6 +58,16 @@ export type BaseData = {
   isPage?: false;
   isAttachment?: false;
   isHome?: boolean;
+
+  // search attributes
+  isSearch?: false;
+  searchQuery?: string;
+
+  // pagination atributes (for Archive)
+  page?: number;
+  path?: string;
+  next?: string;
+  previous?: string;
```

Comment:
@luisherranz this is what I'm not 100% sure about.
Should it be necessary to declare all the new properties on the `BaseData` ?

Reply:
I don't think so because you won't access those props until you make sure that that is an archive (using `data.is..`), but not 100% sure.